test/pod.persona.test.ts

```diff
@@ -0,0 +1,229 @@
+  // expect(mockMint).toHaveBeenCalled();
+  expect(mockMint).toHaveBeenCalledWith(
+    expectedPod.admin,
+    expectedPod.id,
+    ethers.constants.HashZero,
+  );
+});
+
+test('Mint via admin persona throws if the signer is not the admin', async () => {
+  mockGetPodFetchersByAddress();
+  const pod = await getPod(expectedPod.name);
+  pod.getPersonas = jest.fn().mockResolvedValue([{ type: 'admin', address: expectedPod.admin }]);
+  const personas = await pod.getPersonas(expectedPod.admin);
+
+  const mockSigner = {
+    getAddress: jest.fn().mockResolvedValueOnce(expectedPod.member),
+  };
+
+  const mockMint = jest.fn();
+  jest.spyOn(utils, 'getMetropolisContract').mockReturnValueOnce({
+    mint: mockMint,
+  });
+
+  await expect(
+    pod.callAsPersona(pod.mintMember, [expectedPod.admin], personas[0], mockSigner),
+  ).rejects.toThrow('Signer was not admin');
+});
+
+test('Mint via member persona creates a proposal', async () => {
+  mockGetPodFetchersByAddress();
+  jest.spyOn(axios, 'post').mockResolvedValueOnce(constructGqlGetUsers([expectedPod.member]));
+  const pod = await getPod(expectedPod.name);
+  pod.getPersonas = jest.fn().mockResolvedValue([{ type: 'member', address: expectedPod.member }]);
+  const personas = await pod.getPersonas(expectedPod.member);
+
+  const create = jest
+    .spyOn(createSafe, 'createSafeTransaction')
+    .mockResolvedValue(getSafeTransactionFixture());
+
+  await pod.callAsPersona(pod.mintMember, [userAddress2], personas[0]);
+
+  expect(create).toHaveBeenCalled();
+});
+
+test('Mint via member persona throws if the persona address is not a member', async () => {
+  mockGetPodFetchersByAddress();
+  jest.spyOn(axios, 'post').mockResolvedValueOnce(constructGqlGetUsers([expectedPod.member]));
+  const pod = await getPod(expectedPod.name);
+  pod.getPersonas = jest.fn().mockResolvedValue([{ type: 'member', address: userAddress2 }]);
+  const personas = await pod.getPersonas(expectedPod.member);
+
+  await expect(pod.callAsPersona(pod.mintMember, [userAddress2], personas[0])).rejects.toThrow(
+    'Sender must be a member of this pod or one of its sub pods',
+  );
+});
+
+test('Mint via adminPodMember persona creates a proposal', async () => {
+  mockGetPodFetchersByAddress({
+    overrideAdmin: '0xa60DDcba1C0160595BA435b7fed98783EC13cE05',
+  });
+  jest
+    .spyOn(axios, 'post')
+    .mockResolvedValueOnce(constructGqlGetUsers(['0xf0C7d25c942264D6F21871c05d3dB3b98344b499']));
+  const create = jest
+    .spyOn(createSafe, 'createSafeTransaction')
+    .mockResolvedValue(getSafeTransactionFixture());
+
+  const pod = await getPod('1-root-as-admin.pod.eth');
+  const personas = await pod.getPersonas('0x85760ef61c0ccB7BCC4C7A0116d80D59D92e736d');
+
+  await pod.callAsPersona(
+    pod.mintMember,
+    [userAddress2],
+    personas[0],
+    '0x85760ef61c0ccB7BCC4C7A0116d80D59D92e736d',
+  );
+  expect(create).toHaveBeenCalled();
+});
```

Comment:
Example of a test that would be easier to follow with variable names that explicitly show what persona they are.